pkgs/top-level/aliases.nix

```diff
@@ -869,18 +869,18 @@ mapAliases ({
   netbox_3_3 = throw "netbox 3.3 series has been removed as it was EOL"; # Added 2023-09-02
   netbox_3_5 = throw "netbox 3.5 series has been removed as it was EOL"; # Added 2024-01-22
   netease-music-tui = throw "netease-music-tui has been removed due to unmaintained by upstream and broken functionality"; # Added 2024-03-03
-  nextcloud26 = throw ''
-    Nextcloud v26 has been removed from `nixpkgs` as the support for is dropped
-    by upstream in 2024-04. Please upgrade to at least Nextcloud v27 by declaring
+  nextcloud27 = throw ''
```

Comment:
To me it seems too early do that. Especially given that each of these is needed to upgrade, I think it's good to have major releases as long as they're supported in tree.

Reply:
Agree with that but probably we could drop it anyways since it's officially EOL in 1 week https://endoflife.date/nextcloud

Reply:
Uhm, I think I confused the nextcloud EOL with the EOL date from a different software, apologies!

Will file a PR that removes it on master now and updates it (still marking it as eol) along with the other updates on 24.05.

Reply:
See https://github.com/NixOS/nixpkgs/pull/322410, https://github.com/NixOS/nixpkgs/pull/322409, https://github.com/NixOS/nixpkgs/pull/322397